tests/config/test_config_parser.py

```diff
@@ -3,14 +3,15 @@
 """
 
 import pytest
-
-
 from archetype_flask_connexion.config.config_parser import YAMLConfigParser
 
 """
-Test for update_from_yaml function.
+Test that update_from_yaml return a string object
 """
-def test_update_from_yaml(monkeypatch):
+
+
+def test_update_from_yaml_should_returns_str(monkeypatch):
+
     instance = YAMLConfigParser()
     config_path = ["tests/config/sample1.yaml"]
```

Comment:
I would define these variables as constants at the top of the file (so you can reuse them), e.g.:
```
CONFIG_PATH_OKAY = ["some/valid/path"]
CONFIG_PATH_MISSING = ["some/invalid/path"]
CONFIG_VAR_EMPTY = []
```